src/ProjectSystemTools/BuildLogging/Model/Backend/BuildSummary.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.VisualStudio.ProjectSystem.Tools.Providers;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Tools.BuildLogging.Model.Backend
+{
+    /// <summary>
+    /// Immutable Type
+    /// </summary>
+    public sealed class BuildSummary : IBuildSummary
+    {
+        public int BuildID { get; }
```

Comment:
In .NET, it's more common to capitalise as "Id", meaning this would be `BuildId`. Similarly the local variables elsewhere as `buildId`.